solr/core/src/java/org/apache/solr/core/SolrConfig.java

```diff
@@ -357,7 +357,7 @@ public static final Version parseLuceneVersionString(final String matchVersion)
       .add(new SolrPluginInfo(IndexSchemaFactory.class, "schemaFactory", REQUIRE_CLASS))
       .add(new SolrPluginInfo(RestManager.class, "restManager"))
       .add(new SolrPluginInfo(StatsCache.class, "statsCache", REQUIRE_CLASS))
-      .add(new SolrPluginInfo(CircuitBreakerManager.class, "circuitBreaker"))
+      .add(new SolrPluginInfo(CircuitBreakerManager.class, "circuitBreaker", MULTI_OK))
```

Comment:
question 1 of 2: If circuit breakers are individually pluggable, is there still a requirement for the circuit breaker manager to be configurable too?

```suggestion
      .add(new SolrPluginInfo(CircuitBreaker.class, "circuitBreaker", MULTI_OK))
```

Reply:
This isnt the CircuitBreakerManager being pluggable -- this is CircuitBreakerManager being the central point of configuration for all circuit breakers. So yes, circuit breakers are individually pluggable, and no, CBM is no longer pluggable. However, CBM still handles all the configuration management for circuit breakers, hence this registration

Reply:
Thanks for clarifying that circuit breakers are individually pluggable but the circuit breaker manager is no longer pluggable!

Would you be comfortable with me adding commits to the pull request branch? I've tried out experimentally to use `CircuitBreaker.class` instead of `CircuitBreakerManager.class` and also some other suggestions more easily shared in code instead of comment form e.g. use of the PluginInfo's built-in enable(d) attribute.

Reply:
Absolutely!

Reply:
Thanks! Added commits, with hopefully more or less helpful commit messages. Let me know what you think.